settings/volare-file-settings-filtered.json

```diff
@@ -0,0 +1,67 @@
+
+{
+  "vessel": {
+    "name"  : "X-Miles",
+    "brand" : "X-Yachts",
+    "type"  : "X-99",
+    "uuid"  : "urn:mrn:signalk:uuid:f46d2b37-6f63-48b0-8713-8920238186ed"
+  },
+
+  "excludeMatchingPaths": [
+    "navigation.courseOverGroundMagnetic",
+    "performance.velocityMadeGood"
+  ],
+
+  "interfaces": {},
+
+  "pipedProviders": [
+    {
+      "id": "nmeaFromFile",
+      "pipeElements": [
+         {
+           "type": "providers/filestream",
+           "options": {
+             "filename": "samples/plaka.log"
+           },
+           "optionMappings": [
+             {
+               "fromAppProperty": "argv.nmeafilename",
+               "toOption": "filename"
+             }
+           ]
+         },
+         {
+           "type": "providers/throttle",
+           "options": {
+              "rate": 500
+           }
+         },
+         {
+           "type": "providers/liner"
+         },
+         {
+            "type": "providers/nmea0183-signalk",
+            "optionMappings": [
+              {
+               "fromAppProperty": "selfId",
+               "toOption": "selfId"
+              },
+              {
+               "fromAppProperty": "selfType",
+               "toOption": "selfType"
+              }
+            ]
+         },
+         {
+          "type": "providers/keys-filter",
+          "optionMappings": [
```

Comment:
Why not use the simpler `options` to specify thr paths inline? This extra indirection makes the example harder to follow.